core-bundle/src/Resources/contao/languages/de/tl_content.xlf

```diff
@@ -931,4 +943,4 @@
       </trans-unit>
     </body>
   </file>
-</xliff>
```

Comment:
We always have new lines at the end of every file. You have to configure this in your IDE settings.

Reply:
OK :)